core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -227,13 +226,28 @@ class Analyzer(
         scalaCompiler.askCompletionsAt(pos(fileInfo, point), maxResults, caseSens)
       } pipeTo sender
     case UsesOfSymbolAtPointReq(file, point) =>
-      import context.dispatcher
-      val response = if (toSourceFileInfo(file).exists()) {
+      if (toSourceFileInfo(file).exists()) {
+        val p = pos(file, point)
+        scalaCompiler.askLoadedTyped(p.source)
+        scalaCompiler.askSymbolFqn(p) match {
+          case Some(fqn) =>
+            indexer forward FindUsages(fqn.fqnString)
+          case None =>
+            sender ! FalseResponse
```

Comment:
The `Project` actor should ask `Analyzer` the type at point and getting the fqn from the type forward the request to `Indexer`, is that what we want to do?